apm-agent-plugins/apm-opentelemetry/apm-opentelemetry-metrics-exporter/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-opentelemetry</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.35.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-opentelemetry-metrics-exporter</artifactId>
```

Comment:
[For Reviewer]
Why is the exporter in a separate project? This is required for the API-bridge that follows in the next PR: There we will shade & embedded an Otel metrics SDK. In order to install our exporter there, we will need to add a shaded copy of the exporter-project as well.